lib/document/validateDocumentFactory.js

```diff
@@ -1,5 +1,5 @@
 const Document = require('./Document');
-const baseDocumentSchema = require('../../schema/base/document');
+const baseDocumentSchema = require('../../schema/document/documentBase');
```

Comment:
Why not just `/base` as you've done with `documentTransition` below?

Reply:
Resolved https://github.com/dashevo/js-dpp/pull/135#discussion_r398018787